packages/terra-core-docs/src/terra-dev-site/doc/status-view/example/Custom.jsx

```diff
@@ -15,6 +15,7 @@ const StatusViewButtons = [
 const Custom = () => (
   <StatusView
     message="Custom Message"
+    aria-live="polite"
```

Comment:
Let's consider an scenario I am having terra-status-view in my project named `projectA` and `projectB` so do you expect me to add aria-live for all the status-view added in `projectA` and `projectB` to make it accessible for screen readers ??